src/ILCompiler.Compiler/src/IL/Stubs/Bootstrap/StaticSymbolField.cs

```diff
@@ -0,0 +1,107 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
```

Comment:
As we have discussed offline, it would be better to get these static ranges via a method call instead. E.g. you can call it `static IntPtr GetModuleSection(int id, out int length)` - make up some ids for sections that you need, and add a C++ implementation that just returns the current static ranges.